app/models/taxon_presenter.js

```diff
@@ -12,31 +11,45 @@ class TaxonPresenter {
   build() {
     return Promise.resolve(this)
       .then(function(presentedTaxon) {
-        return Promise.all([TaxonomyData.get(), presentedTaxon.buildBreadcrumb()])
-          .then(function (promises) {
-            var taxonomyData = promises[0];
-            var breadcrumbs = promises[1];
-
-            presentedTaxon.taxonomyData = taxonomyData;
-            // TODO: The breadcrumbs will return the current item as the last item in the array, but this is currently
-            // handled elsewhere in the prototype. We should remove that special handling, and use the full breadcrumbs
-            // from here. For now, just remove the last item from the array
-            presentedTaxon.breadcrumb = breadcrumbs.slice(0, -1);
-            presentedTaxon.buildTaxon();
-            presentedTaxon.buildParent();
-            presentedTaxon.buildChildren();
-            presentedTaxon.buildContent();
-            presentedTaxon.checkIfPenultimate();
-
-            return presentedTaxon;
+        return presentedTaxon.buildTaxon()
+          .then(function (taxon) {
+            presentedTaxon.taxon = taxon;
+            return Promise.all([
+              presentedTaxon.buildBreadcrumb(),
+              presentedTaxon.buildChildren(),
+              presentedTaxon.buildContent()
+            ])
+              .then(function (promises) {
+                var breadcrumbs = promises[0];
```

Comment:
I think you could change this to something like:

```
.then(function ([breadcrumbs, children, guidanceContent]) {
```

Instead of assigning to variables based on array positions